### DISCUSSION TITLE
Copilot support still working?

### DISCUSSION THREAD

**Discussion by raghur** [abetlen/llama-cpp-python#1034] [Q&A]
Looks to me that copilot requests now go to different path "POST /v1/engines/copilot-codex/completions HTTP/1.1" 200 OK and on the github extension log, there's only errors parsing response.

**Comment by abetlen** [comment]
@raghur they should still be functioning yes, what parsing errors are you getting?

**Comment by raghur** [comment]
With
```
    "github.copilot.advanced": {
        "debug.testOverrideProxyUrl": "http://localhost:8000",
        "debug.overrideProxyUrl": "http://localhost:8000",
        
    }
```
In github extension output
```
[ERROR] [fetch] [2023-12-23T04:28:51.461Z] Unhandled status from server:,404,{"detail":"Not Found"}
[INFO] [default] [2023-12-23T04:29:14.323Z] [fetchCompletions] engine http://localhost:8000/v1/engines/copilot-codex
```

on server
```
 127.0.0.1:33158 - "POST /v1/v1/engines/copilot-codex/completions HTTP/1.1" 404 Not Found
```

I've also tried setting `debug.overrideengine` to "" and  `debug.overrideModel` and that seems to help
lots of 
```
[ERROR] [streamChoices] [2023-12-23T04:31:01.543Z] Error parsing JSON stream data,
[ERROR] [streamChoices] [2023-12-23T04:31:01.694Z] Error parsing JSON stream data,
[ERROR] [streamChoices] [2023-12-23T04:31:01.844Z] Error parsing JSON stream data,
[ERROR] [streamChoices] [2023-12-23T04:31:01.997Z] Error parsing JSON stream data,
```
but eventually some completions are shown in shadow text

**Comment by abasu0713** [comment]
Do I need to sign-up for a Github co-pilot? I can't seem to use it.

**Reply by xudule** [reply]
it seems that the extension requires login before checking for local server. Did you found any solution for this?
